### PR TITLE
Birthday 2

### PR DESCRIPTION
**Pull request by azdevc**

### CODE REVIEW

File: assets/easy_points_integration.css [hunk 1]
```diff
@@ -118,12 +118,45 @@
...
+    margin: 0 auto;
+  } */
 }
 
 .easy-points__input-date {
   width: 100%;
 }
 
+.easy-points__product-points {
+  margin-left: var(--ep-spacer-1);;
```
Comment (NexPB) [inline assets/easy_points_integration.css:143]: Has an extra `;`

File: assets/easy_points_integration.js [hunk 2]
```diff
@@ -566,7 +566,7 @@ var EasyPoints = {
         .forEach(node => node.addEventListener('click', this.onClickResetBtn));
 
       EasyPoints.Selectors.getCheckoutButtonEl(document, true)
-        .forEach(node => node.addEventListener('click', this.onClickSetCoupon));
+        .forEach(node => node.addEventListener('mousedown', this.onClickSetCoupon));
```
Comment (NexPB) [inline assets/easy_points_integration.js:569]: I still want it to be click by default. For edgecases we'll switch it to `mousedown` as a last resort only.

**Review by NexPB — CHANGES_REQUESTED** [review]

File: assets/easy_points_integration.css (hunk 1 above)
Comment (azdevc) [reply assets/easy_points_integration.css]: done

File: assets/easy_points_integration.js (hunk 2 above)
Comment (azdevc) [reply assets/easy_points_integration.js]: I put it back to click for now, implement solution later

**Pull request merged**